### PR TITLE
fix factory configuration widget

### PR DESCRIPTION
**Pull request by olexii4**
Signed-off-by: Oleksii Orel <oorel@codenvy.com>

### CODE REVIEW

**Comment by benoitf** [comment]
what is the issue ? I don't see JIRA issue or description

**Comment by riuvshin** [comment]
AFAIK there is no issue they just found the problem that factory JSON isn't displayed.

**Comment by benoitf** [comment]
this is not described in PR

File: dashboard/src/app/factories/factory-details/configuration-tab/factory-configuration/factory-configuration.controller.js [hunk 1]
```diff
@@ -24,25 +24,29 @@ export class FactoryConfigurationCtrl {
...
+    }, (originFactory) => {
+      this.$timeout(() => {
+        this.updateFactoryContent(originFactory);
+      }, 1000);
     });
   }
 
   //Update the factory content from origin factory.
   updateFactoryContent(originFactory) {
+    if(!originFactory) {
```
Comment (benoitf) [inline dashboard/src/app/factories/factory-details/configuration-tab/factory-configuration/factory-configuration.controller.js]: space

**Comment by riuvshin** [comment]
Agree, I've asked QA team to create an issue.

File: dashboard/src/app/factories/load-factory/load-factory.controller.js [hunk 2]
```diff
@@ -24,7 +24,7 @@ export class LoadFactoryCtrl {
    * Default constructor that is using resource
    * @ngInject for Dependency injection
    */
-  constructor(cheAPI, codenvyAPI, $websocket, $scope, $route, $timeout, $mdDialog, loadFactoryService, lodash, $location, routeHistory) {
+  constructor(cheAPI, codenvyAPI, $websocket, $route, $timeout, $mdDialog, loadFactoryService, lodash, $location, routeHistory) {
```
Comment (benoitf) [inline dashboard/src/app/factories/load-factory/load-factory.controller.js]: not related

**Comment by olexii4** [comment]
done

**Pull request merged**